source/MRMesh/MRObjectVoxels.cpp

```diff
@@ -24,7 +24,7 @@ MR_ADD_CLASS_FACTORY( ObjectVoxels )
 
 constexpr size_t cVoxelsHistogramBinsNumber = 256;
 
-void ObjectVoxels::construct( const SimpleVolume& volume, const ProgressCallback& cb )
+void ObjectVoxels::construct( const SimpleVolume& volume, ProgressCallback cb )
```

Comment:
Why do we change `const ProgressCallback& cb` to `ProgressCallback cb`?

Reply:
fixed